src/style.css

```diff
@@ -1 +1,31 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
```

Comment:
Consider using a fallback font-family for the stars in case the main one doesn't load. This is a good practice to ensure text remains readable across different platforms and devices.